airbyte-cdk/bulk/toolkits/extract-jdbc/src/test/kotlin/io/airbyte/cdk/h2source/H2SourceIntegrationTest.kt

```diff
@@ -100,30 +99,6 @@ class H2SourceIntegrationTest {
         }
     }
 
-    @Test
-    @Disabled
-    fun testReadGlobal() {
-        H2TestFixture().use { h2: H2TestFixture ->
-            val configPojo =
-                H2SourceConfigurationJsonObject().apply {
-                    port = h2.port
-                    database = h2.database
-                    setCursorMethodValue(CdcCursor)
-                    resumablePreferred = false
-                }
-            SyncsTestFixture.testSyncs(
-                configPojo,
-                h2::createConnection,
-                Companion::prelude,
-                "h2source/expected-cdc-catalog.json",
-                "h2source/cdc-catalog.json",
-                SyncsTestFixture.AfterRead.Companion.fromExpectedMessages(
-                    "h2source/expected-messages-global-cold-start.json",
-                ),
-            )
-        }
-    }
-
```

Comment:
We can straight up remove this, at least for the time being. If we merge this PR we should file a ticket tracking reintroducing this test in a way where we mock the CDC toolkit (there is no debezium for h2)